k6/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM grafana/k6
+
+USER root
+RUN apk add --no-cache \
+		curl
+COPY docker-entrypoint.sh /usr/local/bin/
+COPY handleSummary.js .
+RUN chmod +x /usr/local/bin/docker-entrypoint.sh
+RUN chown 12345 /usr/local/bin/docker-entrypoint.sh
```

Comment:
Set the permissions appropriately on the file in Git, you should not need to change either the file permissions or owner in the Dockerfile.

Reply:
Changes made as specified.